control_panel_api/models.py

```diff
@@ -11,10 +11,13 @@
 
 
 class User(AbstractUser):
+    auth0_id = models.CharField(max_length=128, primary_key=True)
     name = models.CharField(max_length=256, blank=True)
 
     teams = models.ManyToManyField('Team', through='TeamMembership')
 
+    REQUIRED_FIELDS = ['email', 'auth0_id']
```

Comment:
`username` should also be required.

Reply:
This is implicit.

When explicitly specified you get:

"ERRORS:
control_panel_api.User: (auth.E002) The field named as the 'USERNAME_FIELD' for a custom user model must not be included in 'REQUIRED_FIELDS'."

Reply:
Right...it's implicit because `username` in the `AbstractUser` class says `username` is `unique` which means can't be blank.

Reply:
Explicit is better than implicit - can we put username in?